### PR TITLE
yeelali14 PR

### PR DESCRIPTION
**Pull request by yeelali14**

### CODE REVIEW

**Review by vim-zz — CHANGES_REQUESTED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -12,6 +12,27 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
...
+
+# EuropeanPaintings
+
+![image](https://user-images.githubusercontent.com/52451294/131232589-06f209d5-ce33-48ff-aa3a-6e74fa0de637.png)
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 12.2.3.
+
+## Development server
+
+1) Install Angular CLI: `npm install -g @angular/cli`
```
Comment (ShakedZrihen) [inline README.md:30]: Please use React instead

**Review by ShakedZrihen — CHANGES_REQUESTED** [review]
angular is so 2016

File: README.md [hunk 2]
```diff
@@ -12,6 +12,27 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
 4) Navigate to `http://localhost:4200/`. The app will automatically reload.
 
 
+## Further help
+
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+
+#PR
```
Comment (slowHand14) [inline README.md:19]: 👍

File: README.md [hunk 3]
```diff
@@ -12,6 +12,27 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
...
+
+
+# EuropeanPaintings
+
+![image](https://user-images.githubusercontent.com/52451294/131232589-06f209d5-ce33-48ff-aa3a-6e74fa0de637.png)
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 12.2.3.
+
+## Development server
```
Comment (slowHand14) [inline README.md:28]: 👍

**Pull request closed without merging**